pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.398</version>
+        <version>1.568</version>
```

Comment:
Why `1.568`? `1.565.3` and `1.580.3` are both LTS that are supposed to be more stable. Also 1.580.1 includes important security fixes. PR LGTM anyway